tests/reliability/test_cluster/test_cluster_logs/test_cluster_worker_logs_order/test_cluster_worker_logs_order.py

```diff
@@ -42,6 +42,7 @@ def dict_to_tree(dict_tree):
 }
 
 
+@pytest.mark.xfail(reason="known cluster log issue due to cluster logging refactor")
```

Comment:
I think that adding the link of the issue where they will be reworked can be useful:
https://github.com/wazuh/wazuh/issues/20162